compiler/src/iree/compiler/Dialect/Util/Analysis/Constant/OpOracle.cpp

```diff
@@ -100,6 +101,12 @@ ConstExprOpInfo ConstExprOpInfo::getForOp(Operation *op) {
     return {};
   }
 
+  // Target-dependent ops are not const-expr.
+  if (isa<IREE::LinalgExt::UpperBoundTileSizeOp,
```

Comment:
we'll really need a trait/interface for this sooner rather than later (as with the other PR) - pulling a linalgext dep into util is sketchy - can you file an issue for this and mark this with a TODO(#) for cleanup? (util is used by every other part of the compiler and every dep added here then is pulled transitively into the entire rest of the compiler, leading to longer recompiles as a change in linalgext invalidates every single file, etc)